bee-network/bee-autopeering/src/config.rs

```diff
@@ -178,8 +209,8 @@ mod tests {
         {
             "enabled": true,
             "bindAddress": "0.0.0.0:14626",
+            "bindAddressV6": "[::]:14626",
```

Comment:
Is it possible that they use the same port ?

Reply:
Yes, I believe that's possible.